src/firebase.js

```diff
@@ -0,0 +1,13 @@
+import firebase from 'firebase'
+const firebaseConfig = {
+    apiKey: "AIzaSyDaYBnf8ScKqVEp-J-9gNlIjUb-LBqyl2w",
+    authDomain: "platepal.firebaseapp.com",
+    projectId: "platepal",
+    storageBucket: "platepal.appspot.com",
+    messagingSenderId: "989624818914",
+    appId: "1:989624818914:web:32dd13ed6cc8e0e6d092bb"
```

Comment:
This is fine but, for security reasons putting API keys in a repo is unsafe because others will be easily able to access it. 
Try reading this article https://medium.com/better-programming/how-to-hide-your-api-keys-c2b952bc07e6 on how to store API keys more safely. The portion about ".env" will be perfectly fine.

Reply:
> Sorry 1 more thing, from this point on try not to mix using Yarn and NPM commands

I'm a little bit confused, I thought I was only using npm. How can you tell/why does it seem like I'm using yarn? Are we all supposed to be using yarn?

Reply:
whoops my previous reply is in the wrong place

Also, is it just the API key? Do I need to hide message sender id or anything? 
One of your other comments was about not pushing the cache file, and for the .env file I had to make a .gitignore. I put the cache file in there too, hopefully that solves the issue.

Reply:
All of the configuration needed for the firebase API should be hidden

Reply:
Yeah I'm pretty sure it must've been an error then. Also, I only hid the key for now so I will do the rest rn

Reply:
I'm not sure if the google login works anymore, seems to take a while loading